tests/test_completion.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import shlex
+
+commands_without_input = shlex.split(
```

Comment:
I suppose it's mostly a matter of preference, but I would simply put it something like this:
```
commands_without_input = [
    quit,
    true,
    false,
    ...etc...
]
```

Either that, or put it all in a single line (not my preference, though). To me, formatting a list like a text paragraph seems to combine the worst of both worlds:
 * hard to read diffs on removal/addition
 * duplicate entries are not easily spotted (case in point: you already have duplicates)
 * to future contributors, it's unclear how they're supposed to order/group things, because it seems arbitrary

It's a minor thing *here*, but I've seen this pattern in a couple of places, so I felt the need to voice my opinion here.

Reply:
I've formatted it to one line per entry, but I'd like to keep the shlex.split().

Reply:
Fair enough :+1: